### DISCUSSION TITLE
Skipping night / Phantom

### DISCUSSION THREAD

**Discussion by charlyb01** [sf-inc/sihywtcamd#26] [Ideas]
CurseForge suggestion from ShinkuR3D: [here](https://www.curseforge.com/minecraft/mc-mods/sihywtcamd?comment=77).

The idea is that skipping night with sleeping makes things a bit too easy. So several ideas were suggested:
* make it cost xp to skip the night, therefor if you wanted to be able to skip the challenge you'd have to be proactive during the day. Maybe just one or two levels or make it dependent on difficulty
* require the bed be in a suitable structure and have a campfire nearby for you to be able to sleep like how Valheim does it

Note that I won't be able to add it soon as I'm busy with work for now but I'll read your suggestions and discuss with you when I have free time =)

**Comment by charlyb01** [comment]
Another good point:
> it could be annoying to have to deal with phantoms since it would be harder to sleep but it does take a few days for them to spawn so it would probably be alright. It's just that on a conceptual level, I don't like the idea of phantoms in the first place; they are built on a mechanic that punishes you for skipping the challenge which doesn't make much sense to me

It's true that it doesn't make sense that they spawn if you forget to skip the night/skip the challenge that is night.
Maybe we could change the way phantoms spawn. For example, they could have a chance to spawn if you try to skip the night (and if they do, you could not skip the night, or not directly after they spawned). But maybe they should still spawn after a few days because you could just never sleep? I don't know.

**Reply by ShinkuRED** [reply]
You could make them the reverse from what they currently are, as in if you sleep too many nights in a row they will appear.
